### PR TITLE
feat: make enclave namespace and network naming deterministic

### PR DESCRIPTION
**Pull request by tedim52**
## Description:
- Limit enclave names to 60 chars
- Make Kubernetes namespace and Docker network names be `kt-ENCLAVENAME`

## Is this change user facing?
YES

### CODE REVIEW

File: api/golang/engine/lib/enclave/consts.go [hunk 1]
```diff
@@ -1,3 +1,3 @@
 package enclave
 
-const AllowedEnclaveNameCharsRegexStr = `^[-A-Za-z0-9._]{1,63}$`
+const AllowedEnclaveNameCharsRegexStr = `^[-A-Za-z0-9._]{1,59}$`
```
Comment (mieubrisse) [inline api/golang/engine/lib/enclave/consts.go]: isn't this 59 chars (not 60)? either is fine with me, but we should be clear

Comment (mieubrisse) [reply api/golang/engine/lib/enclave/consts.go]: maybe should also add a comment explaining _why_ we're making this particular length restriction

**Review by mieubrisse — APPROVED** [review]

**Comment by mieubrisse** [comment]
Fixes #934

File: api/golang/engine/lib/enclave/consts.go (hunk 1 above)
Comment (tedim52) [reply api/golang/engine/lib/enclave/consts.go]: lol facepalm tedi

**Pull request merged**